wp-content/themes/core/assets/pcss/global/color/_variables.pcss

```diff
@@ -0,0 +1,86 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Variables: Color
+ *
+ * NOTE: WordPress outputs their generated CSS properties on the body element.
+ * To associate color properties w/ WP presets, and to allow style overrides
+ * in the FSE Site Editor, CSS properties must be set on the body element.
+ *
+ * ----------------------------------------------------------------------------- */
+
+body {
+	/* -----------------------------------------------------------------------------
+	 * Brand Colors
+	 * ----------------------------------------------------------------------------- */
+
+	--color-brand-primary: var(--wp--preset--color--brand-primary);
+
+	/* -----------------------------------------------------------------------------
+	 * Brand: Secondary
+	 * ----------------------------------------------------------------------------- */
+
+	--color-brand-secondary: var(--wp--preset--color--brand-secondary);
+
+	/* -----------------------------------------------------------------------------
+	 * Neutral: Grey
```

Comment:
Note: I'd like to tie as many color properties here to the theme.json color palette variables as possible so they're connected, but I think it's best to wait and see how the design shakes out further to figure out a strategy for how they should be connected.